### DISCUSSION TITLE
ssl

### DISCUSSION THREAD

**Discussion by DravenWu** [AlistGo/alist#496] [Q&A]
您好，请教一下，我在宝塔里弄的这个，反代后可以域名正常访问，但浏览器网址那里总显示“不安全”看着不舒服，就去阿里云申请了免费的SSL证书，在宝塔里添加并开启强制https，然后网站就进不去了。。。一直在alist的开场动画，过一会底部就弹出network error，弄了好几天了，实在不会。。可以稍稍指点下不，谢谢咯

**Comment by xhofe** [comment]
- #227
